package.json

```diff
@@ -157,7 +157,7 @@
     "cors": "^2.7.1",
     "css-loader": "^1.0.0",
     "cssnano": "^4.1.10",
-    "cypress": "^6.9.1",
+    "cypress": "^7.4.0",
```

Comment:
Was this supposed to be "^7.5.0"?

Reply:
Yeah, I had a typo when I ran the `yarn upgrade`. Opened a PR to fix this: https://github.com/department-of-veterans-affairs/vets-website/pull/17556